### ISSUE TITLE
Radio group in menu has incorrect count of the items 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#15238]
URL: https://fluentsite.z22.web.core.windows.net/components/toolbar/definition
Example: Toolbar can contain a radio group in a menu

Steps:
1. Navigate to the "More menu button"
2. Press Enter key
3. observe narration when menu is open and focus is set to the menu

Actual: 
For VoiceOver it narrate there are 2 items in the menu what is wrong
For NVDA and JAWS it narrates for first part of the menu 1 of 4, 2 of 4, etc...
but then in the last item when user can navigate by arrow key it says 1 of 1. 

Expected:
For NVDA and JAWS I would expect narrates 1 of 5, 2 of 5, etc...
for VoiceOver it needs to count 5 items

Quick HTML analyze:
Looks like one <li> element should have role presentation, it should fix issue
![image](https://user-images.githubusercontent.com/22620150/94239784-9b680080-ff12-11ea-8b16-7542999a3895.png)

**NVDA narration (JAWS has the same narration):**
![image](https://user-images.githubusercontent.com/22620150/94238684-01538880-ff11-11ea-9090-6ce09ae0ba7f.png)

**VoiceOver narration:**
![image](https://user-images.githubusercontent.com/22620150/94238755-1c25fd00-ff11-11ea-892c-806e46996318.png)
